feat(kpi-overview): Update the kpi related docs (#1043)
camunda/camunda-platform-docs: docs/components/optimize/userguide/processes.md

## Overview

The processes section gives you an overview of all the processes and their status on a single page.

> This section also allows you to set a process owner to take responsibility of a process as well as creating time and quality kpis to track process performance. Additionally, a **Management Dashboard** can be found, which integrates data from all processes in one view.

Reviewer: ```suggestion
This section also allows you to set a process owner to take responsibility of a process as well as viewing time and quality KPIs to track process performance. Additionally, a **Management Dashboard** can be found, which integrates data from all processes in one view.
```
small suggestion but it's not possible to create these from the process page is it?

Once an owner is assigned to a process, you can sort the processes page by owner to quickly see which processes have a common owner.

## Set time and quality KPIs

KPIs are single number reports, which have a target set. A report can be configured as a KPI report through the report configuration panel and the classification to a time or quality KPI happens internally in Optimize based on the measure selected.

> Once a report is configured as KPI, its status can be seen on the **Processes Overview** page. Hovering over its status allows previewing more specific information in regard to the KPIs, such as the target set, the actual current value of the single number report and a link to it.

Reviewer: ```suggestion
Once a report is configured as a KPI report, its status can be seen on the **Processes Overview** page. Hovering over its status allows previewing more specific information in regard to the KPIs, such as the target set, the actual current value of the single number report and a link to it. If a user access a report via this link and does not have authorization to view it, they will not be able to see any data in the report view.
```

> Once a report is configured as KPI, its status can be seen on the **Processes Overview** page. Hovering over its status allows previewing more specific information in regard to the KPIs, such as the target set, the actual current value of the single number report and a link to it. If a user access a report via this link and does not have authorization to view it, they will not be able to see any data in the report view.

Reviewer 2: ```suggestion
Once a report is configured as KPI, its status can be seen on the **Processes Overview** page. Hovering over its status allows previewing more specific information in regard to the KPIs, such as the target set, the actual current value of the single number report and a link to it. If a user accesses a report via this link and does not have authorization to view it, they will not be able to see any data in the report view.
```

Once an owner is assigned to a process, you can sort the processes page by owner to quickly see which processes have a common owner.

> ## Set time and quality kpis

Reviewer: ```suggestion
## Set time and quality KPIs
```

> Kpis are single number reports, which have a target set. A report can be configured as kpi through the report configuration panel and the classification to a time or quality kpi happens internally in Optimize based on the measure selected.

Reviewer: ```suggestion
KPIs are single number reports, which have a target set. A report can be configured as a KPI report through the report configuration panel and the classification to a time or quality KPI happens internally in Optimize based on the measure selected.
```

> Once a report is configured as kpi, its status can be seen on the **Processes Overview** page. Hoovering over its status allows previewing more specific information in regard to the kpis, such as the target set, the actual value of the single number report and a link to it.

Reviewer: ```suggestion
Once a report is configured as KPI, its status can be seen on the **Processes Overview** page. Hovering over its status allows previewing more specific information in regard to the KPIs, such as the target set, the actual  current value of the single number report and a link to it.
```

> ![Set time and quality kpis](./img/kpiConfiguration.png)

Reviewer: ```suggestion
![Set time and quality KPIs](./img/kpiConfiguration.png)
```
